integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/AbstractIntegrationTest.java

```diff
@@ -351,4 +351,12 @@ protected SdxCloudStorageRequest getCloudStorageRequest(TestContext testContext)
         }
         return cloudStorage.getRequest();
     }
+
+    protected int getFreeIpaInstanceCountByGroup(TestContext testContext) {
```

Comment:
I think the naming CountByGroup is misleading here. Should be more like:
`getFreeIpaInstanceCountByProdiver`